src/api/components/components.ts

```diff
@@ -428,6 +428,20 @@ class TookSkillTrigger extends ProTrigger {
 
 	public static override new = () => new this();
 }
+class ConsumeTrigger extends ProTrigger {
+	public constructor() {
+		super({
+			name:        'Consume',
+			description: 'Applies skill effects when a player consume an item',
```

Comment:
Just a little grammar mistake here, it should be `when a player consumes`. Other than that, it looks good!